### PR TITLE
boards: arm: Add u-blox EVK-NINA-B1

### PR DESCRIPTION
**Pull request by ghost**
Add support for u-blox EVK-NINA-B1 which uses the nRF52832.

This board is similar to the nRF52dk_nrf52832 with
different pin assignments on the header pins and not having
the debug-in and shield SWD headers.

Tested with blinky, button, and Bluetooth peripheral_hr

Signed-off-by: Bob Recny <bob.recny@u-blox.com>

### CODE REVIEW

File: boards/arm/ubx_evkninab1_nrf52832/Kconfig.defconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# u-blox EVK-NINA-B1 board configuration
+
+# Copyright (c) 2020 u-blox AG
```
Comment (gpaquet85) [inline boards/arm/ubx_evkninab1_nrf52832/Kconfig.defconfig:3]:
```suggestion
# Copyright (c) 2021 u-blox AG
```

Comment (brec-u-blox) [reply boards/arm/ubx_evkninab1_nrf52832/Kconfig.defconfig]: Corrected - Thanks

**Comment by brec-u-blox** [comment]
Just checking status. Thanks.

**Comment by ioannisg** [comment]
Needs rebase

**Review by ioannisg — APPROVED** [review]

**Review by anangl — APPROVED** [review]

**Pull request merged**